libs/vkd3d/command.c

```diff
@@ -18643,6 +18765,7 @@ static void *d3d12_command_queue_submission_worker_main(void *userdata)
 
         case VKD3D_SUBMISSION_WAIT:
             VKD3D_REGION_BEGIN(queue_wait);
+            d3d12_command_queue_wait_staggered_submission(queue);
```

Comment:
As mentioned on Discord, I think if we're staggered, we should wait on the fence on CPU.